zulipterminal/ui_tools/boxes.py

```diff
@@ -535,7 +535,13 @@ def footlinks_view(
             return None
 
         footlinks = []
-        for link, (text, index, show_footlink) in message_links.items():
+        for limit, message_link in enumerate(message_links.items()):
```

Comment:
`limit` feels like a misnomer. Would `index` be better?

Reply:
We are already using `index` for storing `message_link` index. Using `index` will not affect the code but it will definitely be confusing.
Maybe `pos` or `position`?

Reply:
Could we rather not change this line and use the existing `index` for the check? That would be a lot cleaner and would not require https://github.com/zulip/zulip-terminal/pull/780#discussion_r473230293 either.

Reply:
Yeah, that sounds about right.